packages/sdk-vue/vite.config.ts

```diff
@@ -15,7 +15,7 @@ export default defineConfig({
       fileName: 'vue-fusionauth',
     },
     rollupOptions: {
-      external: ['vue'],
+      external: ['vue', 'nuxt'],
```

Comment:
@michaelyons can you give me some details on what this does

Reply:
@dmackinn I added nuxt to the `rollupOptions.external` so it doesn't get bundled with our package while building.